src/components/LocationPage/ChartsHolder.tsx

```diff
@@ -164,155 +169,13 @@ const ChartsHolder = (props: {
-                    chartIdentifier={Metric.POSITIVE_TESTS}
-                    {...shareButtonProps}
-                  />
-                )}
-              </ChartHeaderWrapper>
-              <ChartLocationName>{projection.locationName}</ChartLocationName>
-              <ChartDescription>
-                {positiveTestsStatusText(projection)}
-              </ChartDescription>
-              {isMobile && testPositiveData && (
-                <ShareButtons
-                  chartIdentifier={Metric.POSITIVE_TESTS}
-                  {...shareButtonProps}
-                />
-              )}
-              {testPositiveData && (
-                <>
-                  <ChartPositiveTestRate columnData={testPositiveData} />
-                  <Disclaimer metricName={Metric.POSITIVE_TESTS} />
-                </>
-              )}
-              <ChartHeaderWrapper>
-                <ChartHeader ref={icuUtilizationRef}>
-                  {getMetricName(Metric.HOSPITAL_USAGE)}
-                  <BetaTag>Beta</BetaTag>
-                </ChartHeader>
-                {!isMobile && icuUtilizationData && (
-                  <ShareButtons
-                    chartIdentifier={Metric.HOSPITAL_USAGE}
-                    {...shareButtonProps}
-                  />
-                )}
-              </ChartHeaderWrapper>
-              <ChartLocationName>{projection.locationName}</ChartLocationName>
-              <ChartDescription>
-                {hospitalOccupancyStatusText(projection)}
-              </ChartDescription>
-              {isMobile && icuUtilizationData && (
-                <ShareButtons
-                  chartIdentifier={Metric.HOSPITAL_USAGE}
-                  {...shareButtonProps}
-                />
-              )}
-              {icuUtilizationData && (
-                <>
-                  <ChartICUHeadroom columnData={icuUtilizationData} />
-                  <Disclaimer metricName={Metric.HOSPITAL_USAGE} />
-                </>
-              )}
-              <ChartHeaderWrapper>
-                <ChartHeader ref={contactTracingRef}>
-                  {getMetricName(Metric.CONTACT_TRACING)}
-                  <BetaTag>Beta</BetaTag>
-                </ChartHeader>
-                {!isMobile && contactTracingData && (
-                  <ShareButtons
-                    chartIdentifier={Metric.CONTACT_TRACING}
-                    {...shareButtonProps}
-                  />
-                )}
-              </ChartHeaderWrapper>
-              <ChartLocationName>{projection.locationName}</ChartLocationName>
-              <ChartDescription>
-                {contactTracingStatusText(projection)}
-              </ChartDescription>
-              {isMobile && contactTracingData && (
-                <ShareButtons
-                  chartIdentifier={Metric.CONTACT_TRACING}
-                  {...shareButtonProps}
+              {chartPropsForMap.map(chartProps => (
+                <ChartBlock
```

Comment:
The `ChartBlock` element should have a key here, see https://reactjs.org/docs/lists-and-keys.html#keys

Reply:
oo yes! added